src/main/haskell/kore/src/Kore/ASTVerifier/AttributesVerifier.hs

```diff
@@ -106,38 +102,13 @@ verifySymbolHookAttribute
     -> AttributesVerification atts
     -> Attributes
     -> Either (Error VerifyError) Hook
-verifySymbolHookAttribute indexedModule =
+verifySymbolHookAttribute _ =
```

Comment:
If the IndexedModule argument is, indeed, not needed, you could fully delete it.